### ISSUE TITLE
Video 3: getLeads and leads is undefined 

### ISSUE THREAD

**Issue by justvidyadhar** [bradtraversy/lead_manager_react_django#3]
Im currently working through video 3 and setting up redux and I cant seem to debug a set of error. The files match the tutorial. I've double checked that with this repo as well. 

Can you please help me with this? 

![2019-02-23 21_18_07-cmd - pipenv shell - python manage py runserver admin](https://user-images.githubusercontent.com/13300052/53288638-8d8b5000-37b0-11e9-8f81-8c5398262122.png)

To make it to refer my work, Here's the link to my repo of work so far: 
https://github.com/justvidyadhar/lead-manager

**Comment by LeoSko** [comment]
Okay, after 3 hours of looking into your code and figuring out I got it.
In your **Dashboard.js**
`import { Leads } from './Leads';`
In original Brad's **Dashboard.js**
`import Leads from "./Leads";`

Those brackets make huge difference here, look more description [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) and [there](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import).
Basically, in your code you import **class Leads** instead of result of **connect(mapStateToProps, { getLeads })(Leads);**. This difference in import lead to undefined props that are usually set in [connect()](https://react-redux.js.org/api/connect), that's what error says.

Fix the import and you should be fine.

**Comment by jjgarrett0** [comment]
@LeoSko Thank you for finding that! I have been following along as well but modifying for my own application (essentially replacing "leads" with "stations") and I couldn't figure out what was going wrong. This just saved me hours of frustration!! Thanks so much!

**Comment by NeilFranks** [comment]
@LeoSko dude thank you
